lib/storage.js

```diff
@@ -7,35 +7,18 @@ const assert = require('assert-plus');
 const bedrock = require('bedrock');
 const database = require('bedrock-mongodb');
 const {promisify} = require('util');
-const {BedrockError} = bedrock.util;
-
-// load config defaults
-require('./config');
+const {util: {BedrockError}} = bedrock;
 
 // module API
 const api = {};
 module.exports = api;
 
 const authorizations = api.authorizations = {};
+api.delegations = require('./storage-delegations');
+api.revocations = require('./storage-revocation');
 const zcaps = api.zcaps = {};
 
 bedrock.events.on('bedrock-mongodb.ready', async () => {
-  /* Note: There are two capability collections:
-
-  1. The `authorization` collection is used to store capabilities that are
-  actively authorized for use. The authorizing party (stored as the
-  `controller`) writes these capabilities to this collection after an
-  application ensures they are permitted to do so for a given
-  `invocationTarget`. Typically, the delegator for a given authorization
-  capability is the same party that writes the capability to the
-  `authorization` collection. An application may later check storage for these
-  active capabilities when verifying a capability invocation by querying for
-  the capability by its ID and invocationTarget.
-
-  2. The `zcap` collection is used to store capabilities for an invoker to
-  later invoke.
```

Comment:
This stuff is still true, we shouldn't remove it yet.

Reply:
Fixed.